tools/ci_build/github/azure-pipelines/templates/c-api-cpu.yml

```diff
@@ -562,30 +562,6 @@ jobs:
         python.exe $(Build.SourcesDirectory)\tools\ci_build\github\windows\post_binary_sizes_to_dashboard.py --commit_hash=$(Build.SourceVersion) --size_data_file=binary_size_data.txt --build_project=Lotus --build_id=$(Build.BuildId)
       workingDirectory: '$(Build.BinariesDirectory)'
 
-  - task: AzureFileCopy@3
-    displayName: 'Copy Signed Native NuGet Package to Blob Store'
```

Comment:
The build step requires AzureRM powershell module, but that module is obsoleted and I don't want to install it on our build machines.

Reply:
do we have support for the AzureCLI step?
https://docs.microsoft.com/en-us/cli/azure/storage/blob?view=azure-cli-latest#az_storage_blob_upload

Reply:
Yes. But nobody is consuming packages from there. We have a nuget feed for daily packages. If anyone need the nuget package, they would prefer to fetch the packages from the feed.

Reply:
And even the feed is not often used.